platform/node/src/node_request.cpp

```diff
@@ -52,7 +58,9 @@ void NodeRequest::HandleCallback(const Nan::FunctionCallbackInfo<v8::Value>& inf
 
     // Move out of the object so callback() can only be fired once.
     auto callback = std::move(request->callback);
+    request->callback = {};
     if (!callback) {
+        request->unref();
```

Comment:
Just to make this code safer, does it work if you create a handle and unref a the beginning of the code? So when the handle goes out of scope we no longer have refs.

Reply:
Perhaps my suggestion is overkill :-)

Reply:
I am not sure how you would want this done honestly, could you say more if you want to change this?